### PR TITLE
browse "mini design sprints"

### PR DESCRIPTION
**Pull request by SBBCornell**
The following two pages are ready for some review/discussion:
/category_taxonomy
and
/tb/recent

The new layouts add more structure to the display and systematize hierarchy of the data. /category_taxonomy also makes use of a simple js accordion script to reduce scrolling. Ive pulled in some subtle styles from submission-ui (color, box-shadow treatments, use of dividers) and added some new style elements.

I'm also working on the stats pages (part of ARXIVNG-3025) and on the next round of mobile abs page changes (ARXIVNG-3004, no PR yet) and they will feed into each other. They are a work in progress and I welcome your feedback. 

Admin team: if you'd like to spin up these pages in person just ping me (Jake, let me know if a screenshare would be helpful)

Screen Shot 2020-02-27 at 12 16 45 PM

Screen Shot 2020-02-27 at 12 17 04 PM

### CODE REVIEW

**Comment by mhl10** [comment]
I think these look great and would also encourage soliciting feedback from the admin team, especially with respect to the category taxonomy page.

**Comment by SBBCornell** [comment]
These three stats pages are also updated and ready for review: Today, Submissions, and Downloads. Screenshots are attached for a quick view. One of the pieces of the design puzzle that Im working on is expanding the arXiv color palette, and charts offer a good opportunity for that. We can always backtrack to a conservative blue palette but Im liking the sunshine! Let me know what you think. 

Screen Shot 2020-03-09 at 3 47 52 PM

Screen Shot 2020-03-09 at 3 47 38 PM

Screen Shot 2020-03-09 at 3 47 22 PM

File: browse/templates/stats/monthly_submissions.html [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
 {% block header_h1 %}<h1><a href="{{ url_for('.home') }}">{{ config['BROWSE_SITE_LABEL'] }}</a>  &gt; stats &gt; monthly submission rates</h1>{% endblock %}
 
+{% block content_stats %}
+  {{ super() }}
+  <div class="chart_text">
+    <p>This chart displays the number of new submissions received during each month since {{ arxiv_start_dt.strftime('%B %Y') }} (after {{ "%.1f"|format(arxiv_age_years|float) }} years). Hover over the graph to see the exact count for a given month.</p>
+  </div>
+  <div id="chart_content">
+    <div class="chart_text">
+      <a class="button is-pulled-right" title="download monthly submission CSV data" href="{{ url_for('browse.stats', command='get_monthly_submissions') }}">download CSV</a>
```
Comment (mhl10) [inline browse/templates/stats/monthly_submissions.html]:
```suggestion
      <a class="button is-pulled-right" title="download monthly submission CSV data" href="{{ url_for('browse.stats', command='get_monthly_submissions') }}">Download CSV</a>
```

File: browse/templates/stats/today.html [hunk 2]
```diff
@@ -1,16 +1,31 @@
 {%- extends "stats/base.html" -%}
-
+{% set pagename = 'Todays Hourly Stats' %}
```
Comment (mhl10) [inline browse/templates/stats/today.html]:
```suggestion
{% set pagename = 'Hourly Usage Stats' %}
```

**Comment by jimentwood** [comment]
Stats colors look good to me. I like the obvious Download button.

**Comment by afromme** [comment]
Looks good! I like the new colors.

**Comment by SBBCornell** [comment]
The new main stats page is added, all testing errors are fixed, and some additional a11y fixes are added too (all stats pages are fully compliant). There are a few other small update on my to-do list but if this batch of changes gets the thumbs up I'd like to get them merged with develop and do the next round in its own branch.

Here's a quick screenshot of the new main page:
Screen Shot 2020-03-10 at 3 45 11 PM

**Comment by DavidLFielding** [comment]
Trackback and stats pages look great!

**Pull request merged**